netlify.toml

```diff
@@ -1,3 +1,10 @@
+# set environment variables
+[build.environment]
+  REACT_APP_NETWORK_ID = "1"
+  REACT_APP_NETWORK_URL = "https://mainnet.infura.io/v3/60ab76e16df54c808e50a79975b4779f"
```

Comment:
These variables can be hidden on the Netlify dashboard Environment Variables instead of exposing them on Github

Reply:
good catch, fixed! thanks.